src/frontend/src/pages/CreateWorkPackagePage/CreateWorkPackageFormView.tsx

```diff
@@ -74,6 +77,8 @@ const CreateWorkPackageFormView: React.FC<CreateWorkPackageFormViewProps> = ({
     startDate.setDate(startDate.getDate() + daysUntilNextMonday);
   }
   const query = useQuery();
+  const { data: project } = useSingleProject(validateWBS(wbsNum));
+  const workPackages = project ? project.workPackages : [];
```

Comment:
check if !project return <LoadingIndicator/> plus check if isError error page and isLoading loading indicator